azkaban-common/src/main/java/azkaban/storage/StorageManager.java

```diff
@@ -15,18 +15,22 @@
  *
  */
 
-package com.linkedin.azkaban.spi;
+package azkaban.storage;
 
-import java.net.URI;
+import azkaban.spi.Storage;
+import com.google.inject.Inject;
 
 
-/**
- * This exception is thrown when there is an attempt to create a duplicate storage key via the {@link Storage}
- * interface.
- */
-public class KeyAlreadyExistsException extends StorageException {
-  public KeyAlreadyExistsException(URI key) {
-    super("Storage key already present: " + key);
+public class StorageManager {
```

Comment:
It is designed to manage storage. It is a work in progress.

I envision this to be the single point of contact for all Azkaban APIs to interact with Storage.